com.github.KCC.yml

```diff
@@ -0,0 +1,91 @@
+app-id: com.github.KCC
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: kcc
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+modules:
```

Comment:
The file should be reformatted to follow https://github.com/flathub/flathub/wiki/YAML-Style-Guide